src/tracer/src/cli/handlers/init/macos_windows.rs

```diff
@@ -0,0 +1,58 @@
+#![cfg(any(target_os = "macos", target_os = "windows"))]
+use crate::cli::handlers::info;
+use crate::cli::handlers::init::arguments::FinalizedInitArgs;
+use crate::cli::helper::wait;
+use crate::daemon::client::DaemonClient;
+use crate::process_identification::constants::{PID_FILE, STDERR_FILE, STDOUT_FILE};
+use crate::utils::analytics;
+use crate::utils::analytics::types::AnalyticsEventType;
+use std::fs::File;
+use std::process::{Command, Stdio};
+
+pub fn macos_windows_no_daemonize(
```

Comment:
what's the difference btw this and the linux one? can't we reuse this code for the linux as well?